### ISSUE TITLE
Error: `Cannot find module '../package.json'` from `dist/http_client.js`

### ISSUE THREAD

**Issue by Strajk** [apify/apify-client-js#235]
After upgrading to apify-client v2 (2.2.0 to be specific),
the `const { version } = dynamicRequire('../package.json');` broke our Storybook. 

At first, I thought it's caused by some sorcery in our Webpack config, but I was able to reproduce it even on fresh Storybook example.

Steps to reproduce
either clone https://github.com/Strajk/demos/tree/master/storybook-webpack-apify-client-v2 , or:

```
npx sb init
npm i react react-dom
npm i apify-client

// Add `import { ApifyClient } from 'apify-client';` to e.g. Button.jsx
npm run storybook

// Check console for `Error: Cannot find module '../package.json'`
```

**Comment by Strajk** [comment]
Changing dynamicRequire to basic require in `/node_modules/apify-client/dist/http_client.js` fixes the issue

```
const { version } = (0, utils_1.dynamicRequire)('../package.json');
to
const { version } = require('../package.json');
```

**Comment by mtrunkat** [comment]
Franta: This might be connected with WebPack that may do a tree-shaking which removes this file

**Comment by mnmkng** [comment]
@B4nan @vladfrangu I think this should be an easy fix. Might be worth taking a look at it soon. It could prevent external users from integrating Apify with the client.

**Comment by B4nan** [comment]
Maybe you should be using the browser bundle instead ([which has the version number inlined during build time](https://github.com/apify/apify-client-js/blob/f23c46120fbb9426a4cb68587aad7dc06c8423e5/src/utils.ts#L114-L116))? But it looks like its not generated in recent versions, last version with it is [v2.0.0](https://unpkg.com/browse/apify-client@2.0.0/dist/bundle.js), so probably broken publishing workflow. 

I guess we could change the code to directly require the `package.json`, @vladfrangu do you remember [why you changed it to the dynamic require](https://github.com/apify/apify-client-js/commit/efc7c804cc2e8800394b128a53866eff4f5a7dbf)? 

Otherwise webpack can be configured to include externals, but it needs to be explicit.

**Comment by vladfrangu** [comment]
Main reason I changed it is because otherwise TS would duplicate the package.json, which is not really ideal. We can move it to a require call I suppose, but then browser bundles will include the full package.json. Your call

**Comment by B4nan** [comment]
Can't we have both? Instead of generic `dynamicRequire` helper, we can have `getVersion` which will statically require the `../package.json`?

```ts
function getVersion() {
    if (typeof BROWSER_BUILD !== 'undefined') {
        // eslint-disable-next-line no-undef -- defined by webpack
        return { version: VERSION };
    }

    // eslint-disable-next-line
    return require('../package.json');
}
```

**Comment by vladfrangu** [comment]
That should work, since the method is only called for getting the version anyways. We should go with that (but I still also want to check out why npm publishes don't have the browser bundle..)

**Comment by mtrunkat** [comment]
Side note: In Apify Console we use client in the front-end without a problem so this might be specific to the Storybook setup of Webpack.

**Comment by heymartinadams** [comment]
Running into this using NextJS 13.

**Comment by B4nan** [comment]
Should be fixed in `v2.6.4-beta.13`, please let us know if it works.

**Comment by heymartinadams** [comment]
Super! ⭐  many thanks! 🙏
